usl_pipeline/study_area_uploader/study_area_uploader/transformers/study_area_transformers.py

```diff
@@ -17,6 +17,9 @@
 from usl_lib.transformers import polygon_transformers
 from usl_lib.writers import polygon_writers
 
+"""Default soil class value that is recognized a non-green area."""
```

Comment:
Looks like a # comment rather than a """string"""
I'm having trouble understanding the comment, though. This is the soil type that indicates a non-green area?

Reply:
Fixed